grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -777,12 +777,12 @@ public GrpcService build() {
                 maxInboundMessageSizeBytes,
                 enableUnframedRequests || enableHttpJsonTranscoding);
         if (enableUnframedRequests) {
-            grpcService = new UnframedGrpcService(
-                    grpcService, handlerRegistry,
-                    unframedGrpcErrorHandler != null ? unframedGrpcErrorHandler
-                                                     : UnframedGrpcErrorHandler.of(),
-                    unframedGrpcStatusFunction != null ? unframedGrpcStatusFunction
-                                                       : UnframedGrpcStatusFunction.of());
+            final UnframedGrpcErrorHandler unframedGrpcErrorHandler = firstNonNull(
+                    this.unframedGrpcErrorHandler, UnframedGrpcErrorHandler.of());
```

Comment:
Updated `UnframedGrpcErrorHandler` too for the same reason.

Reply:
+1, But could you address the comment to apply `orElse` if `unframedGrpcErrorHandler != UnframedGrpcErrorHandler.of()`

Reply:
Do you mean, if `unframedGrpcErrorHandler != UnframedGrpcErrorHandler.of()`, then apply `UnframedGrpcStatusFunction.orElse()` here..?
I couldn't understand it exactly yet 😭

Reply:
Yes, it is.
The main purpose of `orElse(UnframedGrpcStatusMappingFunction.of())` is to prevent `UnframedGrpcStatusFunction` from returning a null value.
Although we allow returing null from the `@Nullable` `UnframedGrpcStatusFunction.apply()`,
the following code line does not expect a null `HttpStatus`.
https://github.com/line/armeria/pull/3948/files#diff-e4a2f87a91362fd158f50fb15908d3e6878ce1bf6db39b364b991c6685b94237R79
```java
final HttpStatus httpStatus = statusFunction.apply(ctx, status, cause);
```
The previous approach did in `ServerBuilder` might help you understand. 
https://github.com/line/armeria/blob/d67e6f1e3b6e8233c1210629b5f9e3ffd1bf5a59/core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java#L1803-L1807

Reply:
(online discussion.) You mean to use `unframedGrpcStatusMappingFunction.orElse()` for not `UnframedGrpcStatusMappingFunction.of()`(default) case. Understood!